grpc/generated/protocol/v0/weaviate_grpc.pb.go

```diff
@@ -26,6 +26,11 @@ import (
 // Requires gRPC-Go v1.32.0 or later.
 const _ = grpc.SupportPackageIsVersion7
 
+const (
+	Weaviate_Search_FullMethodName       = "/weaviategrpc.Weaviate/Search"
+	Weaviate_BatchObjects_FullMethodName = "/weaviategrpc.Weaviate/BatchObjects"
+)
+
```

Comment:
I can see that there are changes in generated proto stubs, but I don't see any changes made to proto files. I'm wondering if this has an impact on the gRPC API... If there are no changes made to `proto files` then can we revert those gRPC proto stubs changes? so that it will be the same as on `master` branch?

Reply:
If I don't run swagger it doesn't pass tests

Reply:
ok, then can you run this script and commit whatever comes out of it?
```
./tools/dev/grpc_regenerate.sh
```

Reply:
@donomii I will revert those files and push a commit with a fix for this into your branch